avb/Android.mk

```diff
@@ -55,6 +55,10 @@ ifneq ($(KERNELFLINGER_DISABLE_DEBUG_PRINT),true)
     LOCAL_CFLAGS += -DAVB_ENABLE_DEBUG
 endif
 
+ifeq ($(TARGET_USE_TPM),true)
+    LOCAL_CFLAGS += -DUSE_TPM -DSOFT_FUSE
```

Comment:
"USE_TPM" is already in KERNELFLINGER_CFLAGS. avb/Android.mk need redefine it in LOCAL_CFLAGS

Reply:
Yes, avb/Android.mk will not get the macro from Android.mk.